### ISSUE TITLE
[Bug]: After milvus recovers from proxy pod failure chaos, the compact interface times out

### ISSUE THREAD

**Issue by zhuwenxing** [milvus-io/milvus#37803]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:master-20241118-f2a2fd68-amd64
- Deployment mode(standalone or cluster):cluster
- MQ type(rocksmq, pulsar or kafka):    
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

```
[2024-11-18T09:22:31.414Z] self = <test_all_collections_after_chaos.TestAllCollection object at 0x7facbc4239d0>

[2024-11-18T09:22:31.414Z] collection_name = 'Checker__OrGVP5l3'

[2024-11-18T09:22:31.414Z] 

[2024-11-18T09:22:31.414Z]     @pytest.mark.tags(CaseLabel.L1)

[2024-11-18T09:22:31.414Z]     def test_milvus_default(self, collection_name):

[2024-11-18T09:22:31.414Z]         self._connect()

[2024-11-18T09:22:31.414Z]         # create

[2024-11-18T09:22:31.414Z]         name = collection_name if collection_name else cf.gen_unique_str("Checker_")

[2024-11-18T09:22:31.414Z]         t0 = time.time()

[2024-11-18T09:22:31.414Z]         schema = Collection(name=name).schema

[2024-11-18T09:22:31.414Z]         collection_w = self.init_collection_wrap(name=name, schema=schema)

[2024-11-18T09:22:31.414Z]         tt = time.time() - t0

[2024-11-18T09:22:31.414Z]         assert collection_w.name == name

[2024-11-18T09:22:31.414Z]         # get collection info

[2024-11-18T09:22:31.414Z]         schema = collection_w.schema

[2024-11-18T09:22:31.414Z]         dim = cf.get_dim_by_schema(schema=schema)

[2024-11-18T09:22:31.414Z]         int64_field_name = cf.get_int64_field_name(schema=schema)

[2024-11-18T09:22:31.414Z]         float_vector_field_name = cf.get_float_vec_field_name(schema=schema)

[2024-11-18T09:22:31.414Z]         float_vector_field_name_list = cf.get_float_vec_field_name_list(schema=schema)

[2024-11-18T09:22:31.414Z]         # compact collection before getting num_entities

[2024-11-18T09:22:31.414Z]         collection_w.flush(timeout=180)

[2024-11-18T09:22:31.414Z] >       collection_w.compact()

[2024-11-18T09:22:31.414Z] 

[2024-11-18T09:22:31.414Z] testcases/test_all_collections_after_chaos.py:45: 

[2024-11-18T09:22:31.414Z] _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

[2024-11-18T09:22:31.414Z] ../utils/wrapper.py:33: in inner_wrapper

[2024-11-18T09:22:31.414Z]     res, result = func(*args, **kwargs)

[2024-11-18T09:22:31.414Z] ../base/collection_wrapper.py:346: in compact

[2024-11-18T09:22:31.414Z]     check_result = ResponseChecker(res, func_name, check_task, check_items, check, **kwargs).run()

[2024-11-18T09:22:31.414Z] ../check/func_check.py:34: in run

[2024-11-18T09:22:31.414Z]     result = self.assert_succ(self.succ, True)

[2024-11-18T09:22:31.414Z] _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

[2024-11-18T09:22:31.414Z] 

[2024-11-18T09:22:31.414Z] self = <check.func_check.ResponseChecker object at 0x7facbc290940>

[2024-11-18T09:22:31.414Z] actual = False, expect = True

[2024-11-18T09:22:31.414Z] 

[2024-11-18T09:22:31.414Z]     def assert_succ(self, actual, expect):

[2024-11-18T09:22:31.414Z] >       assert actual is expect, f"Response of API {self.func_name} expect {expect}, but got {actual}"

[2024-11-18T09:22:31.414Z] E       AssertionError: Response of API compact expect True, but got False

[2024-11-18T09:22:31.414Z] 

[2024-11-18T09:22:31.414Z] ../check/func_check.py:116: AssertionError

[2024-11-18T09:22:31.414Z] ------------------------------ Captured log setup ------------------------------

[2024-11-18T09:22:31.414Z] [2024-11-18 09:19:17 - INFO - ci_test]: *********************************** setup *********************************** (client_base.py:47)

[2024-11-18T09:22:31.414Z] [2024-11-18 09:19:17 - INFO - ci_test]: pymilvus version: 2.5.0rc119 (client_base.py:48)

[2024-11-18T09:22:31.414Z] [2024-11-18 09:19:17 - INFO - ci_test]: [setup_method] Start setup test case test_milvus_default. (client_base.py:49)

[2024-11-18T09:22:31.414Z] ------------------------------ Captured log call -------------------------------

[2024-11-18T09:22:31.414Z] [2024-11-18 09:19:17 - DEBUG - ci_test]: (api_request)  : [Connections.connect] args: ['default', '', '', 'default', ''], kwargs: {'host': '10.255.57.180', 'port': 19530} (api_request.py:62)

[2024-11-18T09:22:31.414Z] [2024-11-18 09:19:17 - DEBUG - ci_test]: (api_response) : None  (api_request.py:37)

[2024-11-18T09:22:31.414Z] [2024-11-18 09:19:17 - INFO - ci_test]: server version: f2a2fd6 (client_base.py:166)

[2024-11-18T09:22:31.414Z] [2024-11-18 09:19:17 - DEBUG - ci_test]: (api_request)  : [Connections.has_connection] args: ['default'], kwargs: {} (api_request.py:62)

[2024-11-18T09:22:31.414Z] [2024-11-18 09:19:17 - DEBUG - ci_test]: (api_response) : True  (api_request.py:37)

[2024-11-18T09:22:31.414Z] [2024-11-18 09:19:17 - DEBUG - ci_test]: (api_request)  : [Collection] args: ['Checker__OrGVP5l3', {'auto_id': False, 'description': '', 'fields': [{'name': 'int64', 'description': '', 'type': <DataType.INT64: 5>, 'is_primary': True, 'auto_id': False}, {'name': 'float', 'description': '', 'type': <DataType.FLOAT: 10>}, {'name': 'varchar', 'description': '', 'type': <DataType......, kwargs: {'consistency_level': 'Strong'} (api_request.py:62)

[2024-11-18T09:22:31.414Z] [2024-11-18 09:19:17 - DEBUG - ci_test]: (api_response) : <Collection>:

[2024-11-18T09:22:31.414Z] -------------

[2024-11-18T09:22:31.414Z] <name>: Checker__OrGVP5l3

[2024-11-18T09:22:31.414Z] <description>: 

[2024-11-18T09:22:31.414Z] <schema>: {'auto_id': False, 'description': '', 'fields': [{'name': 'int64', 'description': '', 'type': <DataType.INT64: 5>, 'is_primary': True, 'auto_id': False}, {'name': 'float', 'description': '', 'type': <DataType.FLOAT: 10>}......  (api_request.py:37)

[2024-11-18T09:22:31.414Z] [2024-11-18 09:19:17 - DEBUG - ci_test]: (api_request)  : [Collection.flush] args: [], kwargs: {'timeout': 180} (api_request.py:62)

[2024-11-18T09:22:31.414Z] [2024-11-18 09:19:18 - DEBUG - ci_test]: (api_response) : None  (api_request.py:37)

[2024-11-18T09:22:31.414Z] [2024-11-18 09:19:18 - DEBUG - ci_test]: (api_request)  : [Collection.compact] args: [False, 180], kwargs: {} (api_request.py:62)

[2024-11-18T09:22:31.414Z] [2024-11-18 09:22:18 - ERROR - pymilvus.decorators]: grpc RpcError: [compact], <_MultiThreadedRendezvous: StatusCode.DEADLINE_EXCEEDED, Deadline Exceeded>, <Time:{'RPC start': '2024-11-18 09:19:18.308560', 'gRPC error': '2024-11-18 09:22:18.311983'}> (decorators.py:151)

[2024-11-18T09:22:31.414Z] [2024-11-18 09:22:18 - ERROR - ci_test]: Traceback (most recent call last):

[2024-11-18T09:22:31.414Z]   File "/home/jenkins/agent/workspace/tests/python_client/utils/api_request.py", line 32, in inner_wrapper

[2024-11-18T09:22:31.415Z]     res = func(*args, **_kwargs)

[2024-11-18T09:22:31.415Z]   File "/home/jenkins/agent/workspace/tests/python_client/utils/api_request.py", line 63, in api_request

[2024-11-18T09:22:31.415Z]     return func(*arg, **kwargs)

[2024-11-18T09:22:31.415Z]   File "/usr/local/lib/python3.8/dist-packages/pymilvus/orm/collection.py", line 1515, in compact

[2024-11-18T09:22:31.415Z]     self.compaction_id = conn.compact(

[2024-11-18T09:22:31.415Z]   File "/usr/local/lib/python3.8/dist-packages/pymilvus/decorators.py", line 155, in handler

[2024-11-18T09:22:31.415Z]     raise e from e

[2024-11-18T09:22:31.415Z]   File "/usr/local/lib/python3.8/dist-packages/pymilvus/decorators.py", line 137, in handler

[2024-11-18T09:22:31.415Z]     return func(*args, **kwargs)

[2024-11-18T09:22:31.415Z]   File "/usr/local/lib/python3.8/dist-packages/pymilvus/decorators.py", line 176, in handler

[2024-11-18T09:22:31.415Z]     return func(self, *args, **kwargs)

[2024-11-18T09:22:31.415Z]   File "/usr/local/lib/python3.8/dist-packages/pymilvus/decorators.py", line 90, in handler

[2024-11-18T09:22:31.415Z]     raise e from e

[2024-11-18T09:22:31.415Z]   File "/usr/local/lib/python3.8/dist-packages/pymilvus/decorators.py", line 86, in handler

[2024-11-18T09:22:31.415Z]     return func(*args, **kwargs)

[2024-11-18T09:22:31.415Z]   File "/usr/local/lib/python3.8/dist-packages/pymilvus/client/grpc_handler.py", line 1600, in compact

[2024-11-18T09:22:31.415Z]     response = future.result()

[2024-11-18T09:22:31.415Z]   File "/usr/local/lib/python3.8/dist-packages/grpc/_channel.py", line 881, in result

[2024-11-18T09:22:31.415Z]     raise self

[2024-11-18T09:22:31.415Z] grpc._channel._MultiThreadedRendezvous: <_MultiThreadedRendezvous of RPC that terminated with:

[2024-11-18T09:22:31.415Z] 	status = StatusCode.DEADLINE_EXCEEDED

[2024-11-18T09:22:31.415Z] 	details = "Deadline Exceeded"

[2024-11-18T09:22:31.415Z] 	debug_error_string = "UNKNOWN:Deadline Exceeded {created_time:"2024-11-18T09:22:18.311571675+00:00", grpc_status:4}"

[2024-11-18T09:22:31.415Z] >

[2024-11-18T09:22:31.415Z]  (api_request.py:45)

[2024-11-18T09:22:31.415Z] [2024-11-18 09:22:18 - ERROR - ci_test]: (api_response) : <_MultiThreadedRendezvous of RPC that terminated with:

[2024-11-18T09:22:31.415Z] 	status = StatusCode.DEADLINE_EXCEEDED

[2024-11-18T09:22:31.415Z] 	details = "Deadline Exceeded"

[2024-11-18T09:22:31.415Z] 	debug_error_string = "UNKNOWN:Deadline Exceeded {created_time:"2024-11-18T09:22:18.311571675+00:00", grpc_status:4}"

[2024-11-18T09:22:31.415Z] > (api_request.py:46)
```

### Milvus Log

failed job: https://qa-jenkins.milvus.io/blue/organizations/jenkins/chaos-test-cron/detail/chaos-test-cron/19522/pipeline
log:
[artifacts-proxy-pod-failure-19522-server-logs.tar.gz](https://github.com/user-attachments/files/17810392/artifacts-proxy-pod-failure-19522-server-logs.tar.gz)

cluster: 4am
ns: chaos-testing
pod info
```

[2024-11-18T09:15:41.453Z] + kubectl get pods -o wide

[2024-11-18T09:15:41.454Z] + grep proxy-pod-failure-19522

[2024-11-18T09:15:42.014Z] proxy-pod-failure-19522-etcd-0                                    1/1     Running            0                  36m     10.104.18.212   4am-node25   <none>           <none>

[2024-11-18T09:15:42.014Z] proxy-pod-failure-19522-etcd-1                                    1/1     Running            0                  36m     10.104.21.211   4am-node24   <none>           <none>

[2024-11-18T09:15:42.014Z] proxy-pod-failure-19522-etcd-2                                    1/1     Running            0                  36m     10.104.17.12    4am-node23   <none>           <none>

[2024-11-18T09:15:42.014Z] proxy-pod-failure-19522-milvus-datanode-87888c79d-778tv           1/1     Running            2 (36m ago)        36m     10.104.16.70    4am-node21   <none>           <none>

[2024-11-18T09:15:42.014Z] proxy-pod-failure-19522-milvus-datanode-87888c79d-t4c92           1/1     Running            2 (36m ago)        36m     10.104.21.202   4am-node24   <none>           <none>

[2024-11-18T09:15:42.014Z] proxy-pod-failure-19522-milvus-indexnode-7bb8b8f59c-hdz7w         1/1     Running            2 (36m ago)        36m     10.104.20.20    4am-node22   <none>           <none>

[2024-11-18T09:15:42.014Z] proxy-pod-failure-19522-milvus-indexnode-7bb8b8f59c-rklxn         1/1     Running            2 (36m ago)        36m     10.104.30.201   4am-node38   <none>           <none>

[2024-11-18T09:15:42.014Z] proxy-pod-failure-19522-milvus-indexnode-7bb8b8f59c-sc76v         1/1     Running            1 (36m ago)        36m     10.104.4.44     4am-node11   <none>           <none>

[2024-11-18T09:15:42.014Z] proxy-pod-failure-19522-milvus-mixcoord-758cbb7489-94rt9          1/1     Running            2 (36m ago)        36m     10.104.30.199   4am-node38   <none>           <none>

[2024-11-18T09:15:42.014Z] proxy-pod-failure-19522-milvus-proxy-586fb44987-65f5g             1/1     Running            5 (9m49s ago)      36m     10.104.5.118    4am-node12   <none>           <none>

[2024-11-18T09:15:42.014Z] proxy-pod-failure-19522-milvus-querynode-b4f5756bc-6jjhs          1/1     Running            2 (36m ago)        36m     10.104.5.117    4am-node12   <none>           <none>

[2024-11-18T09:15:42.014Z] proxy-pod-failure-19522-milvus-querynode-b4f5756bc-7thhb          1/1     Running            2 (36m ago)        36m     10.104.15.14    4am-node20   <none>           <none>

[2024-11-18T09:15:42.014Z] proxy-pod-failure-19522-milvus-querynode-b4f5756bc-t9sns          1/1     Running            2 (36m ago)        36m     10.104.24.215   4am-node29   <none>           <none>

[2024-11-18T09:15:42.014Z] proxy-pod-failure-19522-minio-0                                   1/1     Running            0                  36m     10.104.17.9     4am-node23   <none>           <none>

[2024-11-18T09:15:42.014Z] proxy-pod-failure-19522-minio-1                                   1/1     Running            0                  36m     10.104.18.217   4am-node25   <none>           <none>

[2024-11-18T09:15:42.014Z] proxy-pod-failure-19522-minio-2                                   1/1     Running            0                  36m     10.104.21.210   4am-node24   <none>           <none>

[2024-11-18T09:15:42.014Z] proxy-pod-failure-19522-minio-3                                   1/1     Running            0                  36m     10.104.16.72    4am-node21   <none>           <none>

[2024-11-18T09:15:42.014Z] proxy-pod-failure-19522-pulsar-bookie-0                           1/1     Running            0                  36m     10.104.18.213   4am-node25   <none>           <none>

[2024-11-18T09:15:42.014Z] proxy-pod-failure-19522-pulsar-bookie-1                           1/1     Running            0                  36m     10.104.21.213   4am-node24   <none>           <none>

[2024-11-18T09:15:42.014Z] proxy-pod-failure-19522-pulsar-bookie-init-ctg65                  0/1     Completed          0                  36m     10.104.18.203   4am-node25   <none>           <none>

[2024-11-18T09:15:42.014Z] proxy-pod-failure-19522-pulsar-broker-0                           1/1     Running            0                  36m     10.104.5.119    4am-node12   <none>           <none>

[2024-11-18T09:15:42.014Z] proxy-pod-failure-19522-pulsar-proxy-0                            1/1     Running            0                  36m     10.104.21.203   4am-node24   <none>           <none>

[2024-11-18T09:15:42.014Z] proxy-pod-failure-19522-pulsar-pulsar-init-94lpk                  0/1     Completed          0                  36m     10.104.1.8      4am-node10   <none>           <none>

[2024-11-18T09:15:42.014Z] proxy-pod-failure-19522-pulsar-recovery-0                         1/1     Running            0                  36m     10.104.21.204   4am-node24   <none>           <none>

[2024-11-18T09:15:42.014Z] proxy-pod-failure-19522-pulsar-zookeeper-0                        1/1     Running            0                  36m     10.104.17.10    4am-node23   <none>           <none>
```

**Comment by weiliu1031** [comment]
![image](https://github.com/user-attachments/assets/acd2a883-0494-41d3-810d-3f4aff84f75c)
datacoord doesn't respond to manual compact before context done

**Comment by zhuwenxing** [comment]
failed job: https://qa-jenkins.milvus.io/blue/organizations/jenkins/chaos-test-kafka-cron/detail/chaos-test-kafka-cron/18588/pipeline
log:
[artifacts-datanode-pod-failure-18588-server-logs.tar.gz](https://github.com/user-attachments/files/17823980/artifacts-datanode-pod-failure-18588-server-logs.tar.gz)

image tag: `master-20241119-fceff6ed-amd64`

reproduced in datanode pod failure chaos test

```

[2024-11-19T08:47:07.334Z] [2024-11-19 08:43:54 - DEBUG - ci_test]: (api_request)  : [Connections.connect] args: ['default', '', '', 'default', ''], kwargs: {'host': '10.255.18.14', 'port': 19530} (api_request.py:62)

[2024-11-19T08:47:07.334Z] [2024-11-19 08:43:54 - DEBUG - ci_test]: (api_response) : None  (api_request.py:37)

[2024-11-19T08:47:07.334Z] [2024-11-19 08:43:54 - INFO - ci_test]: server version: fceff6e (client_base.py:166)

[2024-11-19T08:47:07.334Z] [2024-11-19 08:43:54 - DEBUG - ci_test]: (api_request)  : [Connections.has_connection] args: ['default'], kwargs: {} (api_request.py:62)

[2024-11-19T08:47:07.334Z] [2024-11-19 08:43:54 - DEBUG - ci_test]: (api_response) : True  (api_request.py:37)

[2024-11-19T08:47:07.334Z] [2024-11-19 08:43:54 - DEBUG - ci_test]: (api_request)  : [Collection] args: ['Checker__2eW7Y9Zj', {'auto_id': False, 'description': '', 'fields': [{'name': 'int64', 'description': '', 'type': <DataType.INT64: 5>, 'is_primary': True, 'auto_id': False}, {'name': 'float', 'description': '', 'type': <DataType.FLOAT: 10>}, {'name': 'varchar', 'description': '', 'type': <DataType......, kwargs: {'consistency_level': 'Strong'} (api_request.py:62)

[2024-11-19T08:47:07.334Z] [2024-11-19 08:43:54 - DEBUG - ci_test]: (api_response) : <Collection>:

[2024-11-19T08:47:07.334Z] -------------

[2024-11-19T08:47:07.334Z] <name>: Checker__2eW7Y9Zj

[2024-11-19T08:47:07.334Z] <description>: 

[2024-11-19T08:47:07.334Z] <schema>: {'auto_id': False, 'description': '', 'fields': [{'name': 'int64', 'description': '', 'type': <DataType.INT64: 5>, 'is_primary': True, 'auto_id': False}, {'name': 'float', 'description': '', 'type': <DataType.FLOAT: 10>}......  (api_request.py:37)

[2024-11-19T08:47:07.334Z] [2024-11-19 08:43:54 - DEBUG - ci_test]: (api_request)  : [Collection.flush] args: [], kwargs: {'timeout': 180} (api_request.py:62)

[2024-11-19T08:47:07.334Z] [2024-11-19 08:43:55 - DEBUG - ci_test]: (api_response) : None  (api_request.py:37)

[2024-11-19T08:47:07.334Z] [2024-11-19 08:43:55 - DEBUG - ci_test]: (api_request)  : [Collection.compact] args: [False, 180], kwargs: {} (api_request.py:62)

[2024-11-19T08:47:07.334Z] [2024-11-19 08:46:55 - ERROR - pymilvus.decorators]: grpc RpcError: [compact], <_MultiThreadedRendezvous: StatusCode.DEADLINE_EXCEEDED, Deadline Exceeded>, <Time:{'RPC start': '2024-11-19 08:43:55.248173', 'gRPC error': '2024-11-19 08:46:55.255659'}> (decorators.py:151)

[2024-11-19T08:47:07.334Z] [2024-11-19 08:46:55 - ERROR - ci_test]: Traceback (most recent call last):

[2024-11-19T08:47:07.334Z]   File "/home/jenkins/agent/workspace/tests/python_client/utils/api_request.py", line 32, in inner_wrapper

[2024-11-19T08:47:07.334Z]     res = func(*args, **_kwargs)

[2024-11-19T08:47:07.334Z]   File "/home/jenkins/agent/workspace/tests/python_client/utils/api_request.py", line 63, in api_request

[2024-11-19T08:47:07.334Z]     return func(*arg, **kwargs)

[2024-11-19T08:47:07.334Z]   File "/usr/local/lib/python3.8/dist-packages/pymilvus/orm/collection.py", line 1515, in compact

[2024-11-19T08:47:07.334Z]     self.compaction_id = conn.compact(

[2024-11-19T08:47:07.334Z]   File "/usr/local/lib/python3.8/dist-packages/pymilvus/decorators.py", line 155, in handler

[2024-11-19T08:47:07.334Z]     raise e from e

[2024-11-19T08:47:07.334Z]   File "/usr/local/lib/python3.8/dist-packages/pymilvus/decorators.py", line 137, in handler

[2024-11-19T08:47:07.334Z]     return func(*args, **kwargs)

[2024-11-19T08:47:07.334Z]   File "/usr/local/lib/python3.8/dist-packages/pymilvus/decorators.py", line 176, in handler

[2024-11-19T08:47:07.334Z]     return func(self, *args, **kwargs)

[2024-11-19T08:47:07.334Z]   File "/usr/local/lib/python3.8/dist-packages/pymilvus/decorators.py", line 90, in handler

[2024-11-19T08:47:07.334Z]     raise e from e

[2024-11-19T08:47:07.334Z]   File "/usr/local/lib/python3.8/dist-packages/pymilvus/decorators.py", line 86, in handler

[2024-11-19T08:47:07.334Z]     return func(*args, **kwargs)

[2024-11-19T08:47:07.334Z]   File "/usr/local/lib/python3.8/dist-packages/pymilvus/client/grpc_handler.py", line 1600, in compact

[2024-11-19T08:47:07.334Z]     response = future.result()

[2024-11-19T08:47:07.334Z]   File "/usr/local/lib/python3.8/dist-packages/grpc/_channel.py", line 881, in result

[2024-11-19T08:47:07.334Z]     raise self

[2024-11-19T08:47:07.334Z] grpc._channel._MultiThreadedRendezvous: <_MultiThreadedRendezvous of RPC that terminated with:

[2024-11-19T08:47:07.334Z] 	status = StatusCode.DEADLINE_EXCEEDED

[2024-11-19T08:47:07.334Z] 	details = "Deadline Exceeded"

[2024-11-19T08:47:07.334Z] 	debug_error_string = "UNKNOWN:Deadline Exceeded {created_time:"2024-11-19T08:46:55.25452955+00:00", grpc_status:4}"

[2024-11-19T08:47:07.334Z] >

[2024-11-19T08:47:07.334Z]  (api_request.py:45)

[2024-11-19T08:47:07.334Z] [2024-11-19 08:46:55 - ERROR - ci_test]: (api_response) : <_MultiThreadedRendezvous of RPC that terminated with:

[2024-11-19T08:47:07.334Z] 	status = StatusCode.DEADLINE_EXCEEDED

[2024-11-19T08:47:07.334Z] 	details = "Deadline Exceeded"

[2024-11-19T08:47:07.334Z] 	debug_error_string = "UNKNOWN:Deadline Exceeded {created_time:"2024-11-19T08:46:55.25452955+00:00", grpc_status:4}"

[2024-11-19T08:47:07.334Z] > (api_request.py:46)
```

**Comment by yanliang567** [comment]
@weiliu1031 is it ready for verification?

**Comment by zhuwenxing** [comment]
verified and fixed in `master-20241202-cfe56136-amd64`
